### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jtrivinop**
Proposed changes to expand LWK description, make code examples more prominent and add more info on CLI usage

### CODE REVIEW

**Review by delta1 — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -132,6 +92,184 @@ For instance, mobile app devs might be interested mainly in
...
+
+### Rust
+
+#### Rust Examples
+
+* [Create a testnet watch-only wallet from a CT wallet descriptor and get list of transactions](./lwk_wollet/examples/list_transactions.rs)
+
+### Python
+
+#### Build Python wheel
```
Comment (LeoComandini) [inline README.md:105]: you can also install `lwk` from Pypi with `pip install lwk`, I don't know if we want to mention it here

File: README.md [hunk 2]
```diff
@@ -132,6 +92,184 @@ For instance, mobile app devs might be interested mainly in
 While backend developers might want to directly use `lwk_cli`
 in their systems.
 
+## Getting started with LWK Development
+
+### Rust
+
```
Comment (LeoComandini) [inline README.md:98]: Do we expect to put the "build" section here too?

File: README.md [hunk 3]
```diff
@@ -132,6 +92,184 @@ For instance, mobile app devs might be interested mainly in
...
+```sh
+$ lwk_cli signer jade-id
+```
+load Jade using returned ID
+
+```sh
+$ lwk_cli signer load-jade --signer <NAME_FOR_THIS_JADE> --id <ID>
+```
+
+
```
Comment (LeoComandini) [inline README.md:264]: nit: we can remove a few new lines here and below

**Comment by LeoComandini** [comment]
utACK 46e82f28aca1d2d1898124d07866b2d974552a4a with comments

**Pull request closed without merging**